cmd/operator/main.go

```diff
@@ -265,6 +265,13 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "CloudEventSource")
 		os.Exit(1)
 	}
+	if err = (eventingcontrollers.NewClusterCloudEventSourceReconciler(
+		mgr.GetClient(),
+		eventEmitter,
+	)).SetupWithManager(mgr); err != nil {
+		setupLog.Error(err, "unable to create controller", "controller", "ClusterCloudEventSource")
+		os.Exit(1)
+	}
```

Comment:
it might be worthwhile to explore having one just controller with `Watch` on both cluster and namespaced version of the `CloudEventSource`